curve25519_parser/src/lib.rs

```diff
@@ -367,25 +380,25 @@ mod tests {
 
     #[test]
     fn parse_and_check_pubkeys_multi_format() {
-        let pub_key_pem = parse_openssl_ed25519_pubkey(PEM_PUB).unwrap();
-        let pub_key_der = parse_openssl_ed25519_pubkey(DER_PUB).unwrap();
+        let pub_key_pem = parse_openssl_25519_pubkey(PEM_PUB).unwrap();
+        let pub_key_der = parse_openssl_25519_pubkey(ED_DER_PUB).unwrap();
         assert_eq!(pub_key_der.as_bytes().len(), 32);
         assert_eq!(pub_key_der.as_bytes(), pub_key_pem.as_bytes());
-        let priv_key_pem = parse_openssl_ed25519_privkey(PEM_PRIV).unwrap();
-        let priv_key_der = parse_openssl_ed25519_privkey(DER_PRIV).unwrap();
+        let priv_key_pem = parse_openssl_25519_privkey(PEM_PRIV).unwrap();
+        let priv_key_der = parse_openssl_25519_privkey(ED_DER_PRIV).unwrap();
         assert_eq!(priv_key_der.to_bytes().len(), 32);
         assert_eq!(priv_key_der.to_bytes(), priv_key_pem.to_bytes());
     }
 
     #[test]
     fn parse_many_pubkeys() {
-        let pub_keys_pem = parse_openssl_ed25519_pubkeys_pem_many(PEM_PUB).unwrap();
+        let pub_keys_pem = parse_openssl_25519_pubkeys_pem_many(PEM_PUB).unwrap();
         assert_eq!(pub_keys_pem.len(), 1);
-        let pub_key_der = parse_openssl_ed25519_pubkey(DER_PUB).unwrap();
+        let pub_key_der = parse_openssl_25519_pubkey(ED_DER_PUB).unwrap();
         assert_eq!(pub_key_der.as_bytes().len(), 32);
         assert_eq!(pub_key_der.as_bytes(), pub_keys_pem[0].as_bytes());
 
-        let pub_keys_pem = parse_openssl_ed25519_pubkeys_pem_many(PEM_PUB_MANY).unwrap();
+        let pub_keys_pem = parse_openssl_25519_pubkeys_pem_many(PEM_PUB_MANY).unwrap();
```

Comment:
It would be nice to add one of the new x25519 key to the `PEM_PUB_MANY`, and assert it is correctly recovered.
It would ensure that `parse_openssl_25519_pubkeys_pem_many` is able to parse both x25519 and ed25519 keys.